lib/config.js

```diff
@@ -32,16 +32,16 @@ function defaultConfigPaths () {
 function getInCluster () {
   const host = process.env.KUBERNETES_SERVICE_HOST
   const port = process.env.KUBERNETES_SERVICE_PORT
-
+  const root = process.env.RELATIVE_ROOT || '/'
```

Comment:
Could we also set the default to `/var/run/secrets/kubernetes.io/serviceaccount/` and update `caPath`, `tokenPath`, and `namespacePath` to be like `${ KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT }/*`?  I'm guessing that would implement the behavior folks have in mind when they set the `KUBERNETES_CLIENT_SERVICEACCOUNT_ROOT` value.

Reply:
Not sure I fully understand, can you please give an example?

Reply:
The updates you made were what I had in mind. :+1: